run.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/sh
 
-# ensure, that the sync dir exists and is owned by the user
-[ -d $NC_SYNC_DIR ] || mkdir -p $NC_SYNC_DIR
```

Comment:
i removed the 2 lines cause i couldn't found any place where you used the $NC_SYNC_DIR variable. :)